dnSpy/dnSpy.Contracts.DnSpy/Controls/WpfExtensions.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
+
+namespace dnSpy.Contracts.Controls {
+	/// <summary>
+	/// Helper extensions for Wpf Elements
+	/// </summary>
+	public static class WpfExtensions {
```

Comment:
Remove public.

In fact, only one method in this class is used, it can be inlined in that class.